pkg/certificate/service.go

```diff
@@ -50,7 +50,7 @@ func New(ctx context.Context, idCfg *config.IdentityConfig) (daemon.Daemon, erro
 
 	// TODO: This log should be moved to derived-role-cert.go
 	if !idCfg.RoleCert.Use {
-		log.Infof("Role certificate provisioning is disabled with empty options: roles[%s], output directory[%s]", idCfg.RoleCert.TargetDomainRoles, idCfg.RoleCert.Dir)
+		log.Infof("Role certificate provisioning is disabled with empty options: roles[%s], filename format[%s]", idCfg.RoleCert.TargetDomainRoles, idCfg.RoleCert.Format)
```

Comment:
As long as we are aware that we should not have this line changed in this PR, we are good I think.

Reply:
(Or if you want you can make a separate PR, just for the logger)